addons/website/tests/test_create_website.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import patch
+from odoo.tests import HttpCase
+import odoo.tests
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestCreateWebsite(HttpCase):
+
+    def _theme_upgrade_upstream(self):
+        # Actions are quite tricky in this flow.
+        # Because we cannot do _theme_upgrade_upstream
+        # the theme install action isn't consummed
+        # and it puts the client back to choosing a theme
+        # once it just picked one. So we make sure only the
+        # usual action is enabled
```

Comment:
Sorry but what is the goal of writing comments on 55 char line? Wasting space or making it hard to read :yum: ?
Same remark as for everything, don't go over 80 char **but use all the available space up to that**. That is the general rule, only exceptions:
- Commit message, indeed git log (+ tig etc) are inserting 4 spaces after and before your message, so 72 makes it go to 80.
- Code, it is obviously not always possible to respect the 80 rule in python/js/.. code, we just try to fit under the 100 at least

Docstring, however, can and should always use up to 80, no reason not to. 

Also, I'd remove the first sentence `Actions are quite tricky in this flow.`, doesn't bring much value.

Cool to have written a comment, tho :+1: